tests/sentry/tasks/test_data_export.py

```diff
@@ -28,34 +28,13 @@ def test_task_persistent_name(self):
         assert assemble_download.name == "sentry.tasks.data_export.assemble_download"
 
     def test_get_file_name(self):
-        file_name = get_file_name("TESTING", "proj1_user1_test", "ext")
-        assert file_name == "TESTING-proj1_user1_test.ext"
-        file_name = get_file_name("TESTING", "proj1_user1_test")
-        assert file_name == "TESTING-proj1_user1_test.csv"
+        file_name = get_file_name("TESTING", "proj1-user1-test", 1, "ext")
+        assert file_name == "TESTING_proj1-user1-test_1.ext"
+        file_name = get_file_name("TESTING", "proj1-user1-test", 2)
+        assert file_name == "TESTING_proj1-user1-test_2.csv"
 
     def test_issue_by_tag(self):
-        de1 = ExportedData.objects.create(
```

Comment:
No need to test `user` and non-`user` tags here any more, these should be handled in the separate processor tests, whenever I write those